### PR TITLE
CI: Add prebuilt binaries for alpine

### PR DESCRIPTION
**Pull request by orgads**

### CODE REVIEW

**Comment by orgads** [comment]
How do you deploy the binaries to github? Is it done by travis? Should I add alpine there too?

**Comment by orgads** [comment]
@dennisameling @shiftkey Can you please review this?

By the way - Why do you use both travis and github actions?

**Comment by shiftkey** [comment]
> How do you deploy the binaries to github? Is it done by travis? Should I add alpine there too?

Yes, that's the current situation. If you're willing to wire up Travis CI that'll help me get a release out sooner for this.

> By the way - Why do you use both travis and github actions?

It's a transition period and I've not had bandwidth to get this wrapped. https://github.com/atom/node-keytar/issues/320 has been tracking that work but it's been stuck for a while as #334 needs to be merged which is a significant chunk of the remaining work.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ jobs:
       CC: "clang"
       CXX: "clang++"
       npm_config_clang: "1"
+      MAKEFLAGS: -j4
```
Comment (shiftkey) [inline .github/workflows/ci.yml:19]: Is this needed globally? I see the flag is set inside `alpine-release`, which should make this unnecessary if it's only needed for Alpine builds...

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -96,6 +97,36 @@ jobs:
       if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
       env:
         GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+  alpine-release:
```
Comment (shiftkey) [inline .github/workflows/ci.yml:101]: I think we need to add this as a `needs` value for the `npm-publish` job below, which will ensure that we only publish when we have `alpine` prebuilds available.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -96,6 +97,36 @@ jobs:
...
+    runs-on: ubuntu-latest
+    container: node:15-alpine3.12
+    env:
+      MAKEFLAGS: -j4
+    steps:
+      - uses: actions/checkout@v2
+      - name: install build deps
```
Comment (shiftkey) [inline .github/workflows/ci.yml:108]: :art: let's keep these consistent

```suggestion
      - name: Install additional dependencies
```

**Review by shiftkey — COMMENTED** [review]
Thanks for starting this @orgads! I resolved some conflicts so I could see this in action, and I think there's a path forward to getting this ready to merge for our next release...

File: .github/workflows/ci.yml (hunk 1 above)
Comment (orgads) [reply .github/workflows/ci.yml:19]: Removed

File: .github/workflows/ci.yml (hunk 2 above)
Comment (orgads) [reply .github/workflows/ci.yml:101]: Done

File: .github/workflows/ci.yml (hunk 3 above)
Comment (orgads) [reply .github/workflows/ci.yml:108]: Done

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @orgads!

This support will be part of the next release, hopefully on the weekend unless something has gone awry with the rest of the Actions automation in the update I just published.

**Pull request merged**